src/bitcoin-wallet.cpp

```diff
@@ -49,13 +52,13 @@ static void SetupWalletToolArgs(ArgsManager& argsman)
     argsman.AddCommand("createfromdump", "Create new wallet file from dumped records");
 }
 
-static bool WalletAppInit(ArgsManager& args, int argc, char* argv[])
+static int WalletAppInit(ArgsManager& args, int argc, char* argv[])
```

Comment:
Could maybe make this `option<int>`, a categorical difference seems slightly neater to me than reserving a return code for continuing execution.

Reply:
Especially since `EXIT_SUCCESS` and `EXIT_FAILURE` are not guaranteed to be non-`-1` :)